tests/test_pyscalcli.py

```diff
@@ -17,9 +17,6 @@ def test_installed():
     """Test that the command line client is installed in PATH and
     starts up nicely"""
     assert subprocess.check_output(["pyscal", "-h"])
-    assert subprocess.check_output(["pyscal", "--help"])
-
-    assert subprocess.check_output(["pyscal", "--version"])
```

Comment:
These lines do not give any testing value, they only take a lot of time to execute.